server/gamenode/Lobby.ts

```diff
@@ -143,6 +144,26 @@ export class Lobby {
         this.users = [];
     }
 
+    private async fetchCard(cardName: string): Promise<any> {
+        try {
+            const response = await axios.get('https://karabast-assets.s3.amazonaws.com/data/cards/' + encodeURIComponent(cardName));
+            return response.data;
+        } catch (error) {
+            console.error(`Error fetching card: ${cardName}`, error);
+            throw error;
+        }
+    }
+
+    public async setTokens(): Promise<void> {
```

Comment:
Is this an okay time to also implement the initialization for the list of playable card names (e.g., for Regional Governor)? I think Balkhrog is going to work on that really soon so it would be good to go ahead and unblock him. Can be a followup PR if you prefer

Reply:
I can take a look at adding this on my PR since I'm already messing around with game initialization with my current changes.

Reply:
I prefer a follow-up PR for this if that's okay since it'd be cool to have the gameIcons available in the game.